src/devices/ServoMotor/ServoMotor.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.Pwm;
+
+namespace Iot.Device.ServoMotor
+{
+    /// <summary>
+    /// Represents a rotary actuator or linear actuator that allows for precise control of angular or linear position.
+    /// </summary>
+    public class ServoMotor : IDisposable
+    {
+        private PwmChannel _pwmChannel;
+        private readonly int _maximumAngle;
+        private readonly int _maximumPulseWidthMicroseconds;
+        private readonly int _minimumPulseWidthMicroseconds;
+        private readonly double _servoAngleSlope;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ServoMotor"/> class.
+        /// </summary>
+        /// <param name="maximumAngle">The maximum angle the servo motor can move.</param>
+        /// <param name="minimumPulseWidthMicroseconds">The minimum pulse width, in microseconds, that represent an angle for 0 degrees.</param>
+        /// <param name="maximumPulseWidthMicroseconds">The maxnimum pulse width, in microseconds, that represent an angle for maximum angle.</param>
```

Comment:
1. typo in `maximum`
2. `that represent an angle for maximum angle` - huh?

Reply:
`maximumAngle` represents how far the position can travel (from 0 - maximum angle).  It will be 180 degrees in most cases, but I've seen 120/270/360 degrees.  This also allows users to set a smaller range within the define spec if they wanted.  Meaning, they might have a servo that can move 180 degrees, but only want to move with 90 degrees.

In addition, they might want the 90 degrees be a certain range within the 180 degrees.  Meaning, the angle could be set to use 0 - 90 values to represent 45 - 135 values.  They would just need to set the min/max pulse parameters accordingly.

Reply:
@shaggygi I feel like we should call it relative or normalized angle at this point and let it be value from 0-1 - it's usually much easier to operate on: i.e. if you are controlling fuel injection then 1 would be 100%, in any other case where you need specific unit you can just add `MinAngle + RelativeAngle * AngleRange` <-- note how this terminology is much easier to understand - I'm fine with having additional Function/Property returning such result as well

Reply:
@krwq The majority of other servo related APIs i came across use angle in the wording and degrees as the units.  [Here](https://wpilib.screenstepslive.com/s/currentCS/m/java/l/599703-repeatable-low-power-movement-controlling-servos-with-wpilib) is an example.

Maybe we include both SetScale (for 0 - 1) and SetAngle (for degrees).